justspacesapp/templates/create_study.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+  <h2>New Study</h2>
+  <form method="POST" class="post-form">{% csrf_token %}
+     {{ form.as_p }}
```

Comment:
For now, this should work just fine. But, eventually, we should think about the question of [browser-side form validation](https://pageclip.co/blog/2018-02-20-you-should-use-html5-form-validation.html) (particularly given the imminent complexity of our forms). We may need to add `use_required_attribute = False` to the form classes, if we want to prevent browser-side validation, but let's make that determination later. Just raising it here!

Reply:
Definitely! Leaving this open as a future reminder.